main/res/values/strings.xml

```diff
@@ -1586,8 +1586,6 @@
     <string name="init_geokrety_register_ok">Login successful</string>
     <string name="init_geokrety_register_fail">Login failed</string>
     <string name="init_geokrety_description">Authorize c:geo with GeoKrety.org to log GeoKrety trackables.</string>
-    <string name="init_geokrety_cache">Use api.GeoKrety.org</string>
-    <string name="init_geokrety_cache_description">api.Geokrety.org provides cached data for GeoKrety trackables.\nAvailable data contains more GeoKrety informations like \'description\' or \'missing\' status.\n\nData is not live!\nPosition: ~30min.\nDescription: ~24h.</string>
```

Comment:
@rsudev, don't we have some special treatment on string deletion? I can't remember exactly…

Reply:
@kumy Deleting/Editing/Adding in the english strings file is fine. Optionally removals can also be done for the other languages, but anyway Crowdin will sync this on next up/download to Crowdin.